x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/connectors/create_connector/components/manual_configuration.tsx

```diff
@@ -40,9 +59,104 @@ export const ManualConfiguration: React.FC<ManualConfigurationProps> = ({
   const closePopover = () => {
     setPopover(false);
   };
-
+  const { selectedConnector, rawName } = useValues(NewConnectorLogic);
   const [isFlyoutVisible, setIsFlyoutVisible] = useState(false);
   const [flyoutContent, setFlyoutContent] = useState<'manual_config' | 'client'>();
+  const getCodeSnippet = (): string => {
+    const connectorInfo: NativeConnector | undefined = selectedConnector?.serviceType
+      ? NATIVE_CONNECTOR_DEFINITIONS[selectedConnector.serviceType]
+      : undefined;
+    if (!connectorInfo) {
+      return '';
+    }
+
+    const dynamicConfigValues = Object.entries(
+      connectorInfo.configuration as ConnectorConfiguration
+    )
+      .map(([key, config]) => {
+        const defaultValue = config ? JSON.stringify(config.value) : null;
+        return `    "${key}": ${defaultValue}`;
+      })
+      .join(',\n');
+    const CONSOLE_SNIPPET = dedent` # Example of how to create a ${connectorInfo?.name} connector using the API
+# This also creates related resources like an index and an API key.
+# This is an alternative to using the UI creation flow.
+
+# 1. Create an index
+PUT /connector-${rawName}
```

Comment:
No need for leading `/` when creating an index
```suggestion
PUT connector-${rawName}
```

Reply:
Done